idaes/core/process_base.py

```diff
@@ -324,7 +326,27 @@ def _get_stream_table_contents(self, time_point):
         return None
 
     def serialize_contents(self, time_point=0):
-        return self._get_performance_contents(time_point), self._get_stream_table_contents(time_point)
+        """
+        Return the performance contents and stream table
+
+        NOTE: There is the possiblity of a ConfigurationError because 
+        the names of the inlets and outlets of the unit model may not be
+        standard. If this occurs then return an empty dataframe
+
+        Args:
+            time_point: The time
+
+        Returns:
+            performance_contents: Pandas dataframe with the performance contents
+            stream_table: Pandas dataframe with the stream table for a unit model
+        """
+        performance_contents = self._get_performance_contents(time_point)
+        try:
+            stream_table = self._get_stream_table_contents(time_point)
+        except ConfigurationError as err:
+            _log.warning(f"Could not serialize stream table: {err}")
+            stream_table = DataFrame()
```

Comment:
This is more of a general remark than something related to this PR in particular so feel free to disregard it. If this is the only place in the module where `pandas.DataFrame` is used, and only in a subset of cases, it might make sense considering having pandas as a lazy/runtime import by moving the `DataFrame` import here, so that it's imported only if it's necessary to create the fallback `DataFrame`.

This is more a matter of style than performance (since pandas is probably imported somewhere else in `idaes.core`, and when calling the other methods), and this approach has pros and cons to consider.

Reply:
Since the return value of this function is a DataFrame, importing it is, like Thanos, inevitable